src/widgets/breadcrumb/__tests__/__snapshots__/breadcrumb-test.ts.snap

```diff
@@ -28,8 +28,8 @@ exports[`breadcrumb() render renders transformed items correctly 1`] = `
   "refine": [Function],
   "templateProps": {
     "templates": {
-      "home": "Home",
-      "separator": ">",
+      "home": [Function],
+      "separator": [Function],
```

Comment:
These snapshots should go away, but I'd rather do this in a different PR. Applies to all other snapshots of this type.